editor/gui/editor_object_selector.cpp

```diff
@@ -131,6 +132,16 @@ void EditorObjectSelector::update_path() {
 		Ref<Texture2D> obj_icon;
 		if (Object::cast_to<MultiNodeEdit>(obj)) {
 			obj_icon = EditorNode::get_singleton()->get_class_icon(Object::cast_to<MultiNodeEdit>(obj)->get_edited_class_name());
+		} else if (obj->is_class("EditorDebuggerRemoteObject")) {
+			String class_name;
+			if (Ref<Script> script_base = obj->get_script(); script_base.is_valid()) {
```

Comment:
AFAIK, we don't use init statement syntax in the codebase.

Reply:
Sounds good, updated.